content/applications/productivity/iot/config/windows_iot.rst

```diff
@@ -0,0 +1,139 @@
+
+  .. note::
+     Odoo recommends using an updated, recent version of Windows as some older operating systems can
+     cause the Windows virtual :abbr:`IoT (Internet of Things)` to not work.
+
+- Windows computer (laptop, desktop, server)
+- Odoo IoT subscription
+
+  .. seealso::
+     :ref:`iot/iot-eligibility`
+
+Connect the Windows virtual Iot box to an Odoo database
+=======================================================
+
+#. Navigate to the Odoo 16 or higher installation package for Enterprise/Community Windows at
+   `Odoo's Download Page <https://odoo.com/download>`_.
+
+#. Install and setup the Odoo .exe file.
+
+   After the instructions screen click :guilabel:`Next` to start the installation. Agree to the
+   :abbr:`TOS (Terms of Service)`.
+
+#. During the next step of the installation, select :guilabel:`Odoo IoT` from the :guilabel:`Select
+   the type of install` dropdown.
+
+   For reference, the following should be installed:
+
+   - **Odoo server**
+   - **Odoo IoT**
+   - **Nginx WebServer**
+   - **Ghostscript interpreter**
+
+
+   Ensure there is enough space on the computer for the installation and click :guilabel:`Next`.
+
+#. Select the :guilabel:`Destination Folder`.
+
+   .. warning::
+      Odoo's Windows virtual IoT software shouldn't be installed inside any of the Window's User's
+      directories. Doing so won't allow for Nginx to initialize.
+
+   Choosing ``C:\odoo`` as the install location will allow for the Nginx server to start. Click
+   :guilabel:`Install`. If the folder doesn't exist, then create it. Otherwise the installation
+   files will be spread throughout the hard drive.
+
+#. Allow the installation to occur. This may take a few minutes. Click :guilabel:`Next` to continue.
+
+#. Ensure that the :guilabel:`Start Odoo` box is checked and click :guilabel:`Finish`.
+
+#. After installation, the Odoo server will run and automatically open ``localhost:8069`` on your
+   web browser. The webpage should display the :abbr:`IoT (Internet of Things)` Box configuration
+   page.
+
+   .. seealso::
+      A restart of the Windows IoT program may be necessary should the web browser not display
+      anything. :ref:`restart_windows_iot`
+
+#. Connect :abbr:`IoT (Internet of Things)` devices to the Windows computer.
+
+#. Windows should automatically detect the device because the driver is pre-installed on
+   the computer. If not, search for and install the Windows driver for the device.
+
+#. Refresh the :abbr:`IoT (Internet of Things)`  Box Configuration Page and verify the device is
+   seen on the configuration page. If not, reload the handlers through the configuration page.
+
+#. Connect Windows :abbr:`IoT (Internet of Things)` to a database using existing instructions
+   (manually using the Token).
+
+   .. seealso::
+      :doc:`connect`
+
+#. Use the devices connected to :abbr:`IoT (Internet of Things)` as you normally would.
+
+
+Troubleshooting
+===============
+
+.. _restart_windows_iot:
```

Comment:
Let's add the app in the anchor per [the guidelines](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_guidelines.html#prefix-hyperlink-targets-with-application-names)

```suggestion
.. _productivity/restart_windows_iot:
```